.nvmrc

```diff
@@ -1 +1 @@
-12
+13.10.1
```

Comment:
What's the impact of bumping the version? I have the older ones installed currently

Reply:
I just setup the server to install whatever we put in there and figured we should have the latest, but theres no particular reason for it. We could do the LTS version (12.16.1)?